src/logging.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import * as winstonLogger from './Adapters/Logger/WinstonLogger';
+
+let logger = winstonLogger.logger;
+export function setLogger(aLogger) {
+  if (!aLogger) {
+    console.log('NO LOGGER SET!');
+  }
+  logger = aLogger;
+}
+
+function getLogger() {
+  return logger;
+}
+
+export function addTransport(transport) {
```

Comment:
it sure doesn't seem to belong there anymore.  src/Adapters/Logger/WinstonLogger.js handles the transport collection after this pr, so it would seem that the ability to add a transport would go in there if it is to be exposed at all.

there is a likely an easier way to do what i did, but the goal of my change was to allow me to add a transport to the existing logger.

my first thought would be to move additional transports into configureLogger, the transport could be passed as an option.